modules/images/webp-uploads/load.php

```diff
@@ -111,6 +111,102 @@ function webp_uploads_create_sources_property( array $metadata, $attachment_id )
 
 add_filter( 'wp_generate_attachment_metadata', 'webp_uploads_create_sources_property', 10, 2 );
 
+/**
+ * Creates a `sources` property at the root of the metadata to store the information about
+ * the created full sized images. It creates an image with the size of the full image size.
+ *
+ * @since n.e.x.t
+ *
+ * @param array $metadata The current metadta of the modified image.
+ * @param int   $attachment_id The ID of the attachment being modified.
+ * @return array The metadata modified of the image.
+ */
+function webp_uploads_create_sources_property_for_full_image( $metadata, $attachment_id ) {
```

Comment:
this creates the image and updates the sources meta data, right? function name doesn't indicate that, maybe `webp_uploads_create_full_image` or `webp_uploads_create_full_image_and_sources`?

Reply:
:100: I think that's a good idea.